examples/java-lambda-smartapp/README.md

```diff
@@ -89,19 +89,16 @@ First, you'll need to determine the target ARN for the function you just deploye
 function you just deployed in the AWS Console and you will find the ARN in the upper right
 corner.
 
-In the [developer workspace](https://devworkspace.developer.samsung.com/smartthingsconsole/iotweb/site/index.html),
-choose "Automations" on the left navigation bar and create a new WebHook endpoint automation using the following
-settings:
-
-| Option             | Value                               |
-|--------------------|-------------------------------------|
-| SmartApp Instances | Single                              |
-| Hosting Type       | AWS Lambda                          |
-| Target ARN         | The ARN you found above             |
-| Scopes             | `r:devices:*` and `x:devices:*`     |
-| App Settings       | leave blank                         |
-
-Once created, self-publish the automation for testing and you will be ready to test it.
+Then, follow these steps using the
+[developer workspace](https://devworkspace.developer.samsung.com/smartthingsconsole/iotweb/site/index.html):
```

Comment:
The new URL appears to be https://smartthings.developer.samsung.com/workspace/

Reply:
Fixed...as well as the others.